tests/Moq.Tests/Regressions/IssueReportsFixture.cs

```diff
@@ -320,7 +320,7 @@ public interface ISomeInterface
 				void Method();
 			}
 
-			public class ConcreteClass : ISomeInterface
+			public class ClassWithoutParameterlessConstructor : ISomeInterface
```

Comment:
Incorrect class name, see other review comment above.

Reply:
> (P.S. on a side note, in future PRs, I'd appreciate it if you didn't resolve review comments yourself; let the reviewers perform that step. It should be up to the original author of a review comment to decide whether their comment / request has been sufficiently addressed. Letting reviewers resolve comments makes it easier for them to review your PRs because it will take less time to keep an overview of what remains to be done. 😉)

Sorry... I am never sure as of how to resolve conversations. There should be a two-step validation (_Taken into account_ from PR author, _Validated_ from reviewer).

Reply:
Changed in upcoming commit.